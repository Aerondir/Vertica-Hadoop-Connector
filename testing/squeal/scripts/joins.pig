fs -copyFromLocal $script/foo.log joins/foo.log
fs -copyFromLocal $script/bar.log joins/bar.log
w = LOAD 'joins/foo.log' USING PigStorage('|') as (a:int,b:int,c:int,d:chararray);
x = LOAD 'joins/foo.log' USING PigStorage('|') as (a:int,b:int,c:int,d:chararray);
y = LOAD 'joins/bar.log' USING PigStorage('\t') as (d:int,e:int);
z = JOIN x by a LEFT OUTER, w by b;
z2 = JOIN z by w::a, y by d;
STORE z2 INTO 'joins/zzz';
cat joins/zzz;
fs -rmr joins;
