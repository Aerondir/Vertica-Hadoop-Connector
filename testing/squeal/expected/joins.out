10/10/18 23:57:23 INFO squeal.Squeal: Logging error messages to: /home/rvenkatesh/dev/Linux64/Test/squeal_smoke/pigshell.out
Constructing server
Configuring vertica connection
Translating...
Running script schema refresh(SQL)
ROLLBACK: schema "squeal_PigLatin_joins_pig" does not exist
Time: 144ms
Running script sql setup(SQL)
Time: 123ms
Running script test sql(SQL)
Time: 63ms
Running script load3(PIG)
Use GenericOptionsParser for parsing the arguments. Applications should implement Tool for the same.
Total input paths to process : 1
Use GenericOptionsParser for parsing the arguments. Applications should implement Tool for the same.
Total input paths to process : 1
Time: 31612ms
Running script populate vdata4(SQL)
Time: 450ms
Running script store vdata5(PIG)
Use GenericOptionsParser for parsing the arguments. Applications should implement Tool for the same.
Time: 17686ms
Total time: 50252ms
1	1	100	q	1	1	100	q	1	2000
Deleted hdfs://hermetic:9935/user/rvenkatesh/joins
